src/chrome/content/rules/Gametracker.com.xml

```diff
@@ -1,11 +1,7 @@
-<!--
-Disabled by https-everywhere-checker because:
-Fetch error: http://image.www.gametracker.com/ => https://image.www.gametracker.com/: (60, 'SSL certificate problem: certificate has expired')
--->
-<ruleset name="Gametracker.com (partial)" default_off='failed ruleset test'>
+<ruleset name="Gametracker.com (partial)">
```

Comment:
Can you please remove the `(partial)` flag, since the whole site works with https